test-network-function/suite_test.go

```diff
@@ -115,7 +123,16 @@ func TestTest(t *testing.T) {
 	gomega.RegisterFailHandler(ginkgo.Fail)
 	common.SetLogFormat()
 	common.SetLogLevel()
-	log.Info("Version: ", programVersion, " ( ", GitCommit, " )")
+
+	// Display the latest previously released build in case this build is not released
+	// Otherwise display the build version
+	if GitRelease == "" {
```

Comment:
I don't see where "GitRelease" is being initialized, it will always be ``==""``

Reply:
GitRelease is initialized during the builing process, it is a static variable.